string robot_name
string type
---
bool success
string message
